src/renderer/PresetsBar.tsx

```diff
@@ -102,26 +104,35 @@ const PresetsBar = () => {
   // Fetch default presets and custom presets from storage
   useEffect(() => {
     const fetchPresetNames = async () => {
-      const result = await getPresetListFromFiles();
-      // Sort preset names before updating state just in case
-      setPresetNames(result.sort());
+      try {
+        const result = await getPresetListFromFiles();
+        // Sort preset names before updating state just in case
+        setPresetNames(result.sort());
+      } catch (e) {
+        console.error('Failed to get preset list of files');
+        console.error(e);
+        setPresetNames(['']);
```

Comment:
Just curious, is this line necessary? `presetNames` is initialized as `[]` - is there an explicit reason why we want to give it an element when fetching fails?

Reply:
changed to set global error